oap-server/server-starter/src/main/resources/otel-rules/apisix.yaml

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This will parse a textual representation of a duration. The formats
+# accepted are based on the ISO-8601 duration format {@code PnDTnHnMn.nS}
+# with days considered to be exactly 24 hours.
+# <p>
+# Examples:
+# <pre>
+#    "PT20.345S" -- parses as "20.345 seconds"
+#    "PT15M"     -- parses as "15 minutes" (where a minute is 60 seconds)
+#    "PT10H"     -- parses as "10 hours" (where an hour is 3600 seconds)
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+filter: "{ tags -> tags.job_name == 'apisix-monitoring' }" # The OpenTelemetry job name
+expPrefix:  tag({tags -> tags.host_name = 'apisix::' + tags.service_instance_id})
+expSuffix:
+metricPrefix: meter_apisix
+metricsRules:
+  # apisix
+  - name: sv_http_connections
+    exp: apisix_nginx_http_current_connections.sum(['state','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: shared_dict_capacity_bytes
+    exp: apisix_shared_dict_capacity_bytes.sum(['name','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: shared_dict_free_space_bytes
+    exp: apisix_shared_dict_free_space_bytes.sum(['name','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: sv_etcd_indexes
+    exp: apisix_etcd_modify_indexes.sum(['key','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: sv_bandwidth
+    exp: apisix_bandwidth.tagEqual('route','').tagEqual('service','').tagEqual('consumer','').tagEqual('node','').sum(['type','host_name']).rate('PT1M').service(['host_name'] , Layer.APISIX)
+  - name: sv_http_status
+    exp: apisix_http_status.tagEqual('route','').tagEqual('matched_uri','').tagEqual('matched_host','').tagEqual('service','').tagEqual('consumer','').tagEqual('node','').sum(['code','host_name']).rate('PT1M').service(['host_name'] , Layer.APISIX)
```

Comment:
I see the `http_status` from the UI is 0-1，is that correct？
![image](https://user-images.githubusercontent.com/16773043/193461525-c9da58dd-5894-458c-8ac1-4217a6588538.png)

Reply:
may be need more explanations about what this metric meaning?

Reply:
Is this HTTP status or APISIX Server status?

Reply:
It  seems a serious problem , I  would confirm it .

Reply:
> Is this HTTP status or APISIX Server status?

Yes , TYPE is  counter , I  want get  the rate 

```
apisix_http_requests_total 27
# HELP apisix_http_status HTTP status codes per service in APISIX
# TYPE apisix_http_status counter
apisix_http_status{code="404",route="",matched_uri="",matched_host="",service="",consumer="",node=""} 3
apisix_http_status{code="404",route="routes#1",matched_uri="/self/*",matched_host="",service="1",consumer="",node="127.0.0.1"} 3
```

Reply:
![image](https://user-images.githubusercontent.com/3917424/193462607-f35e7d93-274a-42f1-a336-eb885f9dde9b.png)

Reply:
The `different code of http_status per second` ? If  that so, suggest add unit and more explanations on the UI for users to avoid the confusion.

Reply:
change title to `HTTP  status (Rate)` . and Y-axis  distinguish such as `20%` ,`100%`

Reply:
I don't think the unit (Rate) is appropriate, I think `404 per second` not meaning the percentage of the `404`, WDYT @wu-sheng

Reply:
Let's go back to the original, why 404 matters? 

This is a gateway, so, unless the firewall blocks some IPs(such as DDoS), most likely, every client/browser could access any URI randomly. 404 is just a code but doesn't represent the health status.

Reply:
`rate('PT1M')` for a counter only means what the increase/decrease rate of this metric in the past one minute is. It has no logic connection about the percentage of 404 in all requests. Also, as my previous comments, even it is the percentage, 404 actually doesn't have real meaning to measure server performance.

Reply:
Does `apisix_http_status` include all response codes? Or just some >= 300?

Reply:
> I think `404 per second` not meaning the percentage of the `404`

You are right  , I  misunderstood the rate   . In my case , `rate= (http_status_times_upper-http_status_times_lower) / time_diff_in_second ` for every kind  http  status . How  about  choosing `times` as unit ? 

> Let's go back to the original, why 404 matters?

404 dose not matter  , There are two problem 
1. choose an appropriate unit 
2. the screenshot only display 0/1  , it's  strange

Anyway , I  would  test more  times , and  add more case to  screenshot

Reply:
> Does `apisix_http_status` include all response codes? Or just some >= 300?

All code

Reply:
`http_status_times_upper` What does the `upper` suffix mean? `http_status_times` means `counts per HTTP status code`, right?

Reply:
By reading your MAL, this metric should be `HTTP response code trend`. But the graph seems hard to understand, please recheck.

Reply:
https://github.com/apache/skywalking/blob/dd6b7a1fc0f8581e25c86cf6c904848f0c574e92/oap-server/server-starter/src/main/resources/ui-initialized-templates/postgresql/postgresql-service.json#L575-L592
Yes ,I refer to this config , I would rename title to `HTTP status Trend (per second)` .

Reply:
![image](https://user-images.githubusercontent.com/3917424/193541092-dc4a2c7d-7fac-4803-ae4d-3bde2b7feac5.png)